parquet/src/test/java/org/apache/iceberg/avro/TestParquetReadProjection.java

```diff
@@ -45,4 +45,20 @@ protected GenericData.Record writeAndRead(
 
     return Iterables.getOnlyElement(records);
   }
+
+  // Empty struct read is not supported for Parquet
```

Comment:
rather than this comment, this should use proper annotations for each test method to indicate that it is skipped:
```
  @Override
  @Test
  @Disabled("...")
```

Reply:
Done. PR build passed.